AEPIdentity/Tests/TestHelpers/TestableExtensionRuntime.swift

```diff
@@ -83,12 +83,24 @@ class TestableExtensionRuntime: ExtensionRuntime {
     }
 
     func simulateSharedState(extensionName: String, event: Event?, data: (value: [String: Any]?, status: SharedStateStatus)) {
-        otherSharedStates[extensionName] = SharedStateResult(status: data.status, value: data.value)
-        otherSharedStates["\(extensionName)-\(String(describing: event?.id))"] = SharedStateResult(status: data.status, value: data.value)
+        var sharedStateValue : [String: Any]?
```

Comment:
Need your review for this change @cdhoffmann.

Reply:
LGTM